### PR TITLE
Add React.js and npm

### PR DESCRIPTION
**Pull request by marimiyachi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # fridge.ly
 
-## Migrations
+## What we're using
 
-https://github.com/mattes/migrate
+* Migrations: https://github.com/mattes/migrate
+* npm: dependency management
```
Comment (mpuncel) [inline README.md]: Thank you for documenting this

File: Readme [hunk 2]
```diff
@@ -1,5 +0,0 @@
-This directory tree is generated automatically by godep.
```
Comment (mpuncel) [inline Readme]: Thank you for deleting this

File: client/index.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>MARICHAEL</title>
```
Comment (mpuncel) [inline client/index.html]: Thank you for your attention to detail

File: cmd/fridge.ly/main.go [hunk 4]
```diff
@@ -17,11 +17,11 @@ func main() {
 
 	router := gin.New()
 	router.Use(gin.Logger())
-	router.LoadHTMLGlob("templates/*.tmpl.html")
+	router.LoadHTMLGlob("client/*.html")
```
Comment (mpuncel) [inline cmd/fridge.ly/main.go]: Thank you

**Comment by mpuncel** [comment]
Thank you for this work and thank you in advance for merging this because I approve

**Comment by mpuncel** [comment]
Also thanks for doing this!

**Comment by mpuncel** [comment]
And in case I didn't say it yet, thank you

**Comment by marimiyachi** [comment]
youre welcome

File: Rakefile [hunk 5]
```diff
@@ -0,0 +1,14 @@
+namespace :migrate do
+  task :prod do
+    sh "migrate -url $(heroku run 'echo $DATABASE_URL') -path ./migrations up"
+  end
+
+  task :local do
+    database_name = "fridgly_devel"
+    sh "migrate -url postgres://localhost/#{database_name}?sslmode=disable -path ./migrations up"
```
Comment (mpuncel) [inline Rakefile]: don't check this in right

File: client/index.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import {render} from 'react-dom';
+
+class App extends React.Component {
```
Comment (mpuncel) [inline client/index.js]: woot!!

**Comment by mpuncel** [comment]
:ship: once you remove `Rakefile`

**Comment by marimiyachi** [comment]
merged

**Pull request closed without merging**